### PR TITLE
feat: filter products functionality

### PR DESCRIPTION
**Pull request by Prashantrajput11**
products filter functionality: 
includes sorting on basis of  price- low to high
filter on the basis of categories
sort on basis of ratings

### CODE REVIEW

**Review by shashikantmaurya0007 — APPROVED** [review]
Code looks good to be merged

File: src/backend/db/products.js [hunk 1]
```diff
@@ -8,23 +8,122 @@ import { v4 as uuid } from "uuid";
...
+  //   _id: uuid(),
+  //   img: 'https://res.cloudinary.com/df5hhvtdp/image/upload/v1653327347/final-draft_d6zv0s.png',
+  //   title: "facewash",
+  //   price: "225",
+  //   rating: 3.2,
+  //   categoryName: "fiction",
+  // },
+  // {
+  //   _id: uuid(),
+  //   img: 'https://res.cloudinary.com/df5hhvtdp/image/upload/v1653327347/final-draft_d6zv0s.png',
```
Comment (anjali1102) [inline src/backend/db/products.js:67]: can remove and uncomment

File: src/components/ProductCard/ProductCard.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
+.card{
+  color: #fff;
+  /* border: 2px solid green; */
```
Comment (anjali1102) [inline src/components/ProductCard/ProductCard.css:3]: many console in this file

**Review by ashishkatiyar2404 — APPROVED** [review]
the code structure is good,  also code functionality is also good.
you can merge

**Review by anjali1102 — APPROVED** [review]
Liked the checkboxes styling
price range not working
Clear button not working
can align chekbox with their adjacent text
can give gap between radio button and adjacent text
Remove all console.logs

**Pull request merged**